storage/innobase/xtrabackup/doc/source/release-notes/2.2/2.2.11.rst

```diff
@@ -0,0 +1,37 @@
+=============================
+ |Percona XtraBackup| 2.2.11
+=============================
+
+Percona is glad to announce the release of |Percona XtraBackup| 2.2.11 on May 28th 2015. Downloads are available from our download site `here <http://www.percona.com/downloads/XtraBackup/XtraBackup-2.2.11/>`_ and from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories. 
+
+This release is the current stable release in the 2.2 series.
+
+New Features
+------------
+
+ |Percona XtraBackup| has been rebased on |MySQL| 5.6.24.
+
+Bugs Fixed
+----------
+
+ Version check would crash |innobackupex| and abort the backup on *CentOS* 5. Bug fixed :bug:`1255451`.
+
+ |Percona XtraBackup| could crash when preparing the backup taken on |MySQL|/|Percona Server| 5.5. Bug fixed :bug:`1399471` (*Fungo Wang*).
+
+ Taking a long-lasting backup of active databases would fail with ``"Combined size of log files must be < 512 GB"`` error. Bug fixed :bug:`1425269`.
```

Comment:
Backup would went OK. But xtrabackup was unable to prepare a backup with xtrabackup_logfile larger than 512G